crates/autopilot/src/solvable_orders.rs

```diff
@@ -317,15 +320,15 @@ impl SolvableOrdersCache {
     }
 
     async fn fetch_balances(&self, queries: Vec<Query>) -> HashMap<Query, U256> {
+        if self.disable_order_filters {
```

Comment:
Not related to the change, I just saw that this could be an early escape instead of doing work and then returning an empty result.